test/NuGet.Core.Tests/NuGet.Protocol.Tests/Converters/VersionRangeConverterTests.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Newtonsoft.Json;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.Protocol.Plugins.Tests
```

Comment:
seems like the wrong namespace

Reply:
Fixed in https://github.com/NuGet/NuGet.Client/pull/2899/commits/4af3e67799baa5bc5864e06f1226ee9f66bf74b8